### PR TITLE
[US4] - Configure Github Actions for BE

### PR DESCRIPTION
**Pull request by MtTarik**

### CODE REVIEW

**Comment by miegami** [comment]
Please, rename the pull request using the story number and name.

File: .github/workflows/bild.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name: bild
```
Comment (miegami) [inline .github/workflows/bild.yml]: Rename the file to build.yml.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+addons:
+  sonarcloud:
+    organization: "fp3socialnetwork"
+    token:
+      secure: "XmuFuMnAaPP8owxl9DOUBVpx+/CQzIw3bpPCrmdHqFxt5nZMnBdS5j7hH4FRClaJwsPGCPsdtI8Y+AZB89br5yCjYg5upFaw8NWW3VlegXcvuwYrQXVWATb5p1mT9dvQ/GEGo85EtVXjbR6t8VZ8ELndcSvyDTrzLjfZk3Mhm1KgbujdNSDgkzExLdR7DDyGH2ioXox41zO5VoLlJwvdkJM9fHoVyOyQqRdLoEaC6hNhy5B0i6dT7QUcwOAXCjgs3AOuHzVXuAzN97JHZecQE/GaXR/LsrJI6ErDBY0ofAsuvDU022cYeaI8SMVTqwSFZMLypbmfXsh6AyMO2kOjn26SwFfEldMBgd/hj7ZFE5pz90rRsDnL8pgA3yDFwAlZIDcOGk2GeYRjn5r4F+Vyx7cz34hfpJGSAwpkHYei+IV8wXL1/69fjjcUhYUb5ZUHc5gcLVUKtEpGK9psJxyYbsB0fznCAIS6NAL2pOXp3A+VX8GEFwT+hyOP4+ugRdydQXVmwWgEaq9mmZRizcvoPtaQwdjxyZ643DpCZCeG7oKBoYzxikvcKUIW8L6WJz3vIfjESoN8n0aS76V9tpq7GmMOfDiXMPPQuD6TBTCl/tib77/nt9P1tXRCvCsVrIx3zUBXGLzX6TLGAQl94bg5R6Xex5beipbulkqKitDQIbc="
+script:
+  # the following command line builds the project, runs the tests with coverage and then execute the SonarCloud analysis
+  # - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar -Dsonar.projectKey=Leonert_FP3SocialNetwork
+  # mvn clean test jacoco:report coveralls:report
+  # - mvn clean verify sonar:sonar -Pcoverage
```
Comment (miegami) [inline .travis.yml]: Remove commented lines.

**Comment by miegami** [comment]
Resolve merge conflicts and wait until the backend project will be initialized.

File: .github/workflows/bild.yml (hunk 1 above)
Comment (MtTarik) [reply .github/workflows/bild.yml]: I initiated the change and resolved the conflict

File: .travis.yml (hunk 2 above)
Comment (MtTarik) [reply .travis.yml]: I initiated the change and resolved the conflict

**Pull request closed without merging**